types/heic-decode/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for heic-decode 1.1
+// Project: https://github.com/catdad-experiments/heic-decode
+// Definitions by: hieyou1 <https://github.com/hieyou1>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ You can declare types that are available via importing the module */
```

Comment:
```suggestion
```
it comes from template, does it?, if so, get a rid of this instructional remark.